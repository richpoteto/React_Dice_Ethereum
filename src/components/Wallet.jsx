import React from 'react';
import Address from './Address';

function Wallet({ address, network }) {
  return (
    <span>
      Wallet:
      <Address address={address} network={network} />
    </span>
  );
}
Wallet.propTypes = Address.propTypes;

export default Wallet;
