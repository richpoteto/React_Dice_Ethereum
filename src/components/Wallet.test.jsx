import React from 'react';
import renderer from 'react-test-renderer';
import Wallet from './Wallet';

it('renders correctly', () => {
  const tree = renderer
    .create(<Wallet address="0x0123" network={3} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
