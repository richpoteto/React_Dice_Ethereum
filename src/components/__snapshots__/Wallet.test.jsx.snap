// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span>
  Wallet:
  <a
    href="https://ropsten.etherscan.io/address/0x0123"
  >
    0x0123
  </a>
</span>
`;
